docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -198,6 +198,21 @@ We’ve improved the way you can configure annotations by adding the possibility
 
 {{< figure src="/media/docs/grafana/screenshot-grafana-10-0-annotation-filtering.png" max-width="750px" caption="Annotation filtering" >}}
 
+### Redesigned and improved Log Context
+
+Our team has recently made enhancements to Grafana's log context feature, resulting in a more seamless and consistent user experience. With the updated user interface, users can expect the same level of functionality and usability in log context as they would in any other logs panel.
+
+Notably, we've added the follwoing new features that streamline the log context experience:
+
+1. Log details with actions including a "copy" button for easily copying lines and an eye icon for displaying only selected labels, allowing users to focus on specific information without leaving the log context panel.
+2. A "Wrap Lines" toggle to automatically wrap long lines of text for easier reading and analysis of log entry context directly in log context.
+3. To execute the context query for a log entry in a split screen in the Explore view we've added a "Open in split view" button.
```

Comment:
Given the way the enumeration is described ("the following new  features"), maybe:

```suggestion
3. An "Open in split view" button to execute the context query for a log entry in a split screen in the Explore view.
```